packages/db/package.json

```diff
@@ -36,6 +36,9 @@
     "test": "jest --verbose --detectOpenHandles --forceExit"
   },
   "dependencies": {
+    "@graphql-tools/delegate": "^8.4.3",
+    "@graphql-tools/schema": "^8.3.1",
+    "@graphql-tools/utils": "^8.5.5",
```

Comment:
Looks like this can be moved to `devDependencies` (assuming it's only used with `import type`)

Reply:
:+1: